federator/README.md

```diff
@@ -47,22 +52,27 @@ Inside the .json files there is also the host to that network, for example this
 ```
 
 You need to change `"<YOUR NODE HOST AND RPC PORT>"` for the url of your node for that network and the json rpc port. `Remember to do it for both networks`.
-Also you need to create a `federetaros.key` file with the federator private in it.
+Also you need to create a `federetars.key` file with the federator private in it.
```

Comment:
Haven't checked the code, but probably it's `federators.key`.

Reply:
Fixed